airgun/views/virtwho_configure.py

```diff
@@ -195,14 +221,21 @@ def is_displayed(self):
     @View.nested
     class overview(SatTab):
         status = VirtwhoConfigureStatus('.')
+        debug = VirtwhoConfiguresDebug()
```

Comment:
```suggestion
        debug = Text('.//span[contains(@class,'config-debug')]')
```
Just a query , What if you keep the above for debug ? will you not able to see the status in same ?

Reply:
Can't get the debug status by Text(), it debug checkbox is checked, the element is:
`<span class="config-debug"><span class="fa fa-check "></span> </span>`
If unchecked, the html is:
`<span class="config-debug"></span>`

This is why I have to use `VirtwhoConfiguresDebug()` to get the status.